README.md

```
@@ -60,7 +60,7 @@ $ helm install bindizr oci://registry-1.docker.io/kweonminsung/bindizr-chart \
 Brings up Bindizr, PostgreSQL, and BIND9 on an overlay network.
 
 ```bash
-$ docker stack deploy -c docker-compose.yml bindizr
+$ docker stack deploy -c examples/swarm/docker-compose.yml bindizr
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)